tests/utils.py

```diff
@@ -147,7 +147,7 @@ def generate(directory, databricks_cli, context):
     params = {
         **(
             AWS_DEFAULT_PARAMS
-            if context.get("input_cloud") == "aws"
+            if (context.get("input_cloud") == "aws") | (context.get("input_cloud") == "gcp")
```

Comment:
Let's flip it as mentioned above

Reply:
@arpitjasa-db  not clear on the correction here

Reply:
```suggestion
AZURE_DEFAULT_PARAMS
            if context.get("input_cloud") == "azure"
            else AWS_DEFAULT_PARAMS
```